homeassistant/components/script/strings.json

```diff
@@ -64,7 +64,7 @@
     },
     "toggle": {
       "name": "[%key:common::action::toggle%]",
-      "description": "Toggle a script. Starts it, if isn't running, stops it otherwise."
+      "description": "Starts a script if isn't running, stops it otherwise."
```

Comment:
```suggestion
      "description": "Starts a script if it isn't running, stops it otherwise."
```

Reply:
Oh, thanks for the correction. Didn't notice the error when I carried that over from the previous string.